R/pdf_document.R

```diff
@@ -201,7 +202,7 @@ pdf_document <- function(toc = FALSE,
 # Use filter to set pdf geometry defaults (while making sure we don't override
 # any geometry settings already specified by the user)
 pdf_pre_processor <- function(metadata, input_file, runtime, knit_meta, files_dir,
-                              output_dir) {
+                              output_dir, extra_dependencies) {
```

Comment:
Then to prevent the RStudio IDE warning "`extra_dependencies` not in scope", I need to move the `pdf_pre_processor` function into the `pdf_document` function body, just like it is done for the `html_document` case.